lib/http-bridge/test/integration/Main.hs

```diff
@@ -69,13 +71,15 @@ main = hspec $ do
     describe "Cardano.WalletSpec" Wallet.spec
     describe "Cardano.Wallet.HttpBridge.NetworkSpec" HttpBridge.spec
     describe "CLI commands not requiring bridge" MnemonicsCLI.spec
-    beforeAll startCluster $ afterAll killCluster $ after tearDown $ do
+    beforeAll (startCluster (Just 1337)) $ afterAll killCluster $ after tearDown $ do
```

Comment:
Is this `1337` required to be the same number as the previous test cases? If so, perhaps can they all be the same named constant?

Reply:
Fixed, most tests use random port selection, and there is a defaultPort variable for the fixed port tests.